provisioning/datastore.yml

```diff
@@ -21,7 +21,12 @@
   become: true
 
 - name: set default group write ACLs on /srv/datastore
-  command: /usr/bin/setfacl -m d:g::rwx /srv/datastore
```

Comment:
`default`, `permissions`, and `etype` were added in Ansible 1.5, perhaps you made the command before that time?